src/commands/play.rs

```diff
@@ -35,11 +38,12 @@ pub enum Mode {
     Jump,
 }
 
-#[derive(Clone, Copy)]
+#[derive(Clone)]
```

Comment:
It's because we cannot implement Copy for types that contain a String.
This is why we now have to do `query_type.clone()` in a few places now, unfortunately.

Reply:
It is more verbose, but it does the same (calling clone or not calling it with a `Copy` trait)